lucidworks/connectors-sdk-resources
# IMAP Connector

# Tl;dr
Added IMAP Connector

# Details
The IMAP connector downloads and parses email messages from an IMAP server.

# How to verify
- Follow the build instructions
- Upload the connector to the Fusion Blob Store
- Configure a new IMAP data source
- Start the data source job and verify that messages are being downloaded with the following fields: From, To, Cc, Reply To, Subject, Body
